src/model/CommandLineTool/CommandLineToolModel.ts

```diff
@@ -5,22 +5,59 @@ import {CommandLineBinding} from "../../mappings/draft-4/CommandLineBinding";
 import {Expression} from "../../mappings/draft-4/Expression";
 import {CommandInputParameterModel} from "./CommandInputParameterModel";
 import {CommandOutputParameterModel} from "./CommandOutputParameterModel";
-import {CWLCollection} from "./CWLCollection";
-import {CommandOutputParameter} from "../../mappings/draft-4/CommandOutputParameter";
-import {CommandInputParameter} from "../../mappings/draft-4/CommandInputParameter";
 import {CommandLinePart} from "../helpers/CommandLinePart";
+import {JobHelper} from "../helpers/JobHelper";
 
 export class CommandLineToolModel implements CommandLineTool {
-    inputs: Array<CommandInputParameter>;
-    outputs: Array<CommandOutputParameter>;
+    constructor(json: any) {
+        function objToArr(obj) {
```

Comment:
This can be shortened
```javascript
const obj = {
    foo: {moo: "bar"},
    baz: {taz: "gaz"}
};

const arr = Object.keys(obj).map(id => Object.assign(obj[id], {id}));
```

Produces
```javascript
[ { moo: 'bar', id: 'foo' }, { taz: 'gaz', id: 'baz' } ]
```